<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_roupaNovaVenda_finalizar</class>
 <widget class="QDialog" name="fm_roupaNovaVenda_finalizar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finalizar venda</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="btnCadastrar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled{background:#5abec0;}
QPushButton:disabled{background:#2fbec0;}
QPushButton:pressed{background:#73bebe;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Cadastrar novo cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Cliente selecionado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbNome">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineNome">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>150</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTel">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Telefone (Apenas números)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbEnd">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Endereço</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>150</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbRef">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Refêrencia</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineRef">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>150</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCep">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>CEP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCep">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCnpj">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>CNPJ (Apenas números)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCnpj">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}
QLineEdit{
color:blue;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
        </property>
        <property name="maxLength">
         <number>14</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCpf">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>CPF (Apenas números)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCpf">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:blue;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Finalizar venda</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
        </property>
        <property name="text">
         <string>Selecione o metodo de pagamento</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbTipoPgmt">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dinheiro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cartão</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comodato</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Transferência</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>À vista</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbValor">
        <property name="styleSheet">
         <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Valor final: R$ 0.00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vs">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelecionar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled{background:#7dbe50;}
QPushButton:disabled{background:#aaff7f;}
QPushButton:pressed{background:#aaff00;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Finalizar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="glFiltro">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineFiltro">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:3px;
border:1px solid black;
}</string>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnFiltro">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled{background:#7dbe50;}
QPushButton:disabled{background:#aaff7f;}
QPushButton:pressed{background:#aaff00;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Limpar filtro</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lbFiltro">
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e8e8e8;
color:#808080;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Filtro</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="tw">
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
color:#303030;
font-size:9pt;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefone</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineFiltro</tabstop>
  <tabstop>tw</tabstop>
  <tabstop>btnCadastrar</tabstop>
  <tabstop>lineNome</tabstop>
  <tabstop>lineTel</tabstop>
  <tabstop>lineEnd</tabstop>
  <tabstop>lineRef</tabstop>
  <tabstop>lineCep</tabstop>
  <tabstop>lineCnpj</tabstop>
  <tabstop>cbTipoPgmt</tabstop>
  <tabstop>btnSelecionar</tabstop>
  <tabstop>btnFiltro</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
